### PR TITLE
Issue 2296 - Agent in Kube / OpenShift clusters on ppc64le

### LINKED ISSUE #2296: Agent in Kube / OpenShift clusters on ppc64le
Kubernetes / OpenShift clusters are the most popular container runtime provisioner & orchestration environment for enterprise workloads. And this issue, in addition to Docker support, is for enabling agent support for running it in cluster mode on Power (ppc64le) architecture.

### PR DESCRIPTION
**Pull request by jacob-rogers**
Signed-off-by: Dmitry Gorbachev <dmitry.gorbachev@ru.ibm.com>

Resolves #2296

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -384,11 +384,14 @@ agbot-package: agbot-image
...
 anax-k8s-image: anax-k8s-clean
-	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)
-	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)
-	cp -f $(LICENSE_FILE) $(ANAX_K8S_CONTAINER_DIR)
-	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE_STG)"
-	cd $(ANAX_K8S_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(ANAX_K8S_IMAGE_LABELS) -t $(ANAX_K8S_IMAGE_STG) -f Dockerfile.ubi . && \
+	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR);
+	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR);
+	cp -f $(LICENSE_FILE) $(ANAX_K8S_CONTAINER_DIR);
+	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE_STG)";
```
Comment (bmpotter) [inline Makefile:390]: Why did you add `;` to these lines? That seems unnecessary.

File: Makefile [hunk 2]
```diff
@@ -384,11 +384,14 @@ agbot-package: agbot-image
...
+	if [[ $(arch) == "amd64" || $(arch) == "ppc64el" ]]; then \
+		echo "Using image for $(arch) arch"; \
+		cd $(ANAX_K8S_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(ANAX_K8S_IMAGE_LABELS) -t $(ANAX_K8S_IMAGE_STG) -f Dockerfile.ubi.$(arch) .; \
+	fi
```
Comment (bmpotter) [inline Makefile:394]: Is this just adding a debug echo? The echo above this already prints `$(ANAX_K8S_IMAGE_STG)`, which contains `$(arch)`, so why isn't that sufficient?

**Review by bmpotter — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jacob-rogers) [reply Makefile:390]: Sorry, somehow accidentally got there, by autoformat

File: Makefile (hunk 2 above)
Comment (jacob-rogers) [reply Makefile:394]: Thx, will remove it

File: Makefile (hunk 1 above)
Comment (jacob-rogers) [reply Makefile]: fixed

File: Makefile (hunk 2 above)
Comment (jacob-rogers) [reply Makefile]: fixed

**Review by bmpotter — APPROVED** [review]

**Review by dabooz — APPROVED** [review]
